README.md

```diff
@@ -124,6 +124,12 @@ The Flutter package workflow consists of the following steps:
 
 **Default** `"."`
 
+#### `test_optimization`
+
+**Optional** Enable the test optimization.
+
+**Default** `true`
```

Comment:
What about mentioning that this is useful for golden tests, something like this:

```suggestion
**Default** `true`

Note: since the optimization process group tests into a single file, Golden tests will not work with it, so consider disabling the optimization if you are using those kind of tests.
```